op-deployer/pkg/deployer/opcm/read_superchain_deployment.go

```diff
@@ -0,0 +1,30 @@
+package opcm
+
+import (
+	"github.com/ethereum-optimism/optimism/op-chain-ops/script"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type ReadSuperchainDeploymentInput struct {
+	OPCMAddress common.Address `abi:"opcmAddress"`
+}
+
+type ReadSuperchainDeploymentOutput struct {
+	ProtocolVersionsImpl  common.Address
+	ProtocolVersionsProxy common.Address
+	SuperchainConfigImpl  common.Address
+	SuperchainConfigProxy common.Address
+	SuperchainProxyAdmin  common.Address
```

Comment:
Could you inline the standard [SuperchainContracts](https://github.com/ethereum-optimism/optimism/blob/develop/op-chain-ops/addresses/contracts.go#L18-L24) struct here instead of calling out all the contracts individually?